Python/Distance-Calculator-App/main.py

```diff
@@ -0,0 +1,22 @@
+from geopy.geocoders import Nominatim
+from geopy import distance
+import argparse
+def location():
+    geolocator = Nominatim(user_agent="geoapiExercises")
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--firstlocation', type=str, required=True)
```

Comment:
I suppose there must be predefined location values. What if a user enters some random name of location such as `abc`? Please add a try-catch exception handling for the same.

Reply:
@sohamsshah Can you please tell how it can be done? As if there will be predefined location then it will be not a optimistic script :)

Reply:
I mean that what if the user enters a location that doesn't exist. How will you handle that scenario?

Reply:
@sohamsshah Sir, the module geopy works on latitude and longitude, we can apply try catch conditions for locations like `abc` or `xyz` but what if user enters something like `bnefuivbhewgvuiwehrioufvhnw` , so, we can't apply conditions for every possible probability. But for the case something like `fh2qwercbhuyh` the module itself shows error.

Reply:
See [this](https://geopy.readthedocs.io/en/stable/#specifying-parameters-once). The `geocode` accepts the location in string form i.e. the name of the place eg. `india`, `paris` etc. 

Possible fixes:
```python

try:
# put your location fetcher here
except:
print("Invalid location")
```

Something like this should work. Idea is not to throw errors/terminate the program if the user accidentally misspells and if that place doesn't exist, the program will throw error; which we want to handle it. We don't need to check every possibility.  Let me know if any help is needed for implementing this.

Reply:
@sohamsshah Done with changes :)